platform/security/core/security-core-impl/pom.xml

```diff
@@ -273,7 +273,7 @@
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.17</minimum>
+                                            <minimum>0.10</minimum>
```

Comment:
✏️ with coverage so low, it might be worth adding a ticket to jira to come back and improve.